Bots Explained

```diff
@@ -0,0 +1,34 @@
+When my Mentor Oscar Oranagwa asked me to write about bots, I decided to use this opportunity to clarify exactly on what bots
+are, to both my fellow developers as well as any newbie with the desire to benefit from this innovation. I will use a story
+to explain it all to an extent but before that, let me begin with;
+
+WHAT ARE BOTS?
+BOTS are also known as Internet Robots, and just as the name Robot implies; Bots make things easier, they can handle 
+repetitive tasks and many more. Formerly, the popular version of bots came in the form of malware, but that has begun 
+to change with tech companies like Facebook, Telegram, Chatfuel providing a more useful version of this technology, 
+and they represent CHAT BOTS.
+
+WHY ARE THEY SO IMPORTANT?
+Now let me tell you a story about ChatBots and how they came to be in this ever growing internet generation, evolving gradually 
+to this stage, gathering momentum as it approaches this point.
+In the beginning was Mr.Internet, the current King of transmitting information, who ruled with greatness, knowledge and love, 
+and basically has a lot of tricks under his sleeve. One day, he realized an organizational problem in his kingdom, and since
+he is a big fan of delegating and organizing, he appoints worthy Knights to handle this problem, Knighthood meaning Websites.
+So this was where Facebook, Google, Twitter and the rest came in.
+Now these improved the situation in the Kingdom, and things were awesome and his Kingdom (Information) grew larger, and as it 
+grew, things became disorganized once more, and so Mr.Internet called for meeting, discussed the problem and came up with a 
+solution. He said "Why don't you have your own specific kingdoms and territories and mansions where you can receive your guests"
+and they all agreed to this, and then he gave them a condition saying "But you must call these territories, APPS ". And they 
```

Comment:
The story is not accurate but if gives the reader an idea to what is really happening